tools/StaticAnalysis/Exceptions/Az.Resources/BreakingChangeIssues.csv

```diff
@@ -1 +1,73 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
Note: is there a reason we can't publish the nuget package above?

Reply:
The package has been published and is being referenced instead of the local `.nupkg` file.